### PR TITLE
update golangci-lint to 1.60.1

### PR DESCRIPTION
**Pull request by Luap99**
Fixes new spotted issues around printf() formats and using os.Setenv() in tests.

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by Luap99** [comment]
Well this was painful but it is done now and we can keep updating our linter

File: cmd/podman/containers/commit.go
Comment (mheon) [inline cmd/podman/containers/commit.go:109]: Why is this being removed? It looks like it ought to do something

File: hack/golangci-lint.sh
Comment (mheon) [inline hack/golangci-lint.sh:25]: I assume this isn't needed because remote == tunnel?

File: test/e2e/run_privileged_test.go
Comment (Luap99) [reply test/e2e/run_privileged_test.go:141]: yes

File: hack/golangci-lint.sh
Comment (Luap99) [reply hack/golangci-lint.sh:25]: yep, on linux we just lint the same things twice because of that and the linter isn't exactly fast so I dropped it.

File: cmd/podman/containers/commit.go
Comment (Luap99) [reply cmd/podman/containers/commit.go:109]: This thing is called several times in the same code path so I dropped one instance

**Comment by mheon** [comment]
LGTM

**Comment by Luap99** [comment]
Ok this seems to indeed work while other PRs are failing: https://cirrus-ci.com/task/5484764008808448

@containers/podman-maintainers PTAL let's get this in to unblock CI.

**Review by vrothberg — APPROVED** [review]
/lgtm

**Pull request merged**
